deploy/docs/Best_Practices.md

```diff
@@ -0,0 +1,68 @@
+
+# Advanced Configuration / Best Practices
+
+
+### Multiline Log Support
+
+By default, we use a regex that matches the first line of a multiline log starting with a date of the format : `2019-11-17 07:14:12`.
+
+Users can specify their custom regex to detect first line of a multiline log and parse the logs correctly.
+
+New parsers can be defined under the `parsers` key of the fluent-bit configuration section in `values.yaml` as follows:
+
+```
+parsers:
+  enabled: true
+  regex:
+    - name: new_parser_name
```

Comment:
in the example can we keep the current `multi_line` parser (since the customer will likely want to keep this parser and *append* their parser).  So the example should show 2 parsers with the 2nd one being the newly added example.  We should also try to come up with a regex example different from the current one